### PR TITLE
chore(Loader): Convert to RFC

### PR DESCRIPTION
**Pull request by assuncaocharles**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

# BREAKING CHANGES

This PR converts `Loader` component to be functional. Restricting props set that will be passed to styles functions.

Related to #12237

## Prop sets

| `Loader`    |
| --------- |
| `inline` |
| `labelPosition` |
| `size` |

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/fluentui/react-northstar/src/components/Loader/Loader.tsx [hunk 1]
```diff
@@ -60,102 +73,126 @@ export const loaderSlotClassNames: LoaderSlotClassNames = {
...
+  } = props => {
+  const context: ProviderContextPrepared = React.useContext(ThemeContext);
+  const { setStart, setEnd } = useTelemetry(Loader.displayName, context.telemetry);
+  setStart();
+  const { delay, label, indicator, svg, inline, labelPosition, className, design, styles, variables, size } = props;
+
+  const ElementType = getElementType(props);
+  const unhandledProps = useUnhandledProps(Loader.handledProps, props);
+
+  let delayTimer: number;
```
Comment (layershifter) [inline packages/fluentui/react-northstar/src/components/Loader/Loader.tsx:93]:
```suggestion
  const delayTimer = React.useRef<number>();
```

**Comment by layershifter** [comment]
Please fix PR description, props there doesn't match to `mapPropsToStyles`.

File: packages/fluentui/react-northstar/src/components/Loader/Loader.tsx (hunk 1 above)
Comment (assuncaocharles) [reply packages/fluentui/react-northstar/src/components/Loader/Loader.tsx]: Done

**Comment by layershifter** [comment]
> Please fix PR description, props there doesn't match to `mapPropsToStyles`.

Now it's empty, but there should be `inline`, `labelPosition` and `size`.
Please include a changelog entry.

**Pull request merged**